pyproject.toml

```diff
@@ -193,12 +192,16 @@ ignore_missing_imports = true
 
 
 [tool.pytest.ini_options]
+minversion = "8"
```

Comment:
This one is somehow redundant with this previous line:
https://github.com/python-poetry/poetry/blob/b6bad9c73c9e7e74a1a10df22210837a1e231ba1/pyproject.toml#L72

They serve a different purpose though:
- the first line defines which dependencies will be installed for the test target,
- the second line raises an error if an existing older pytest is run on the repository.

I am not sure about this change. Thoughts?

Reply:
I don't see a reason to duplicate it, our dependency spec should be enough